JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2138,7 +2143,8 @@
         warmup            = options$warmup,
         adapt_delta       = options$adapt_delta,
         control           = list(max_treedepth = options$max_treedepth),
-        seed              = if (isTRUE(options[["setSeed"]])) options[["seed"]]
+        seed              = if (isTRUE(options[["setSeed"]])) options[["seed"]],
```

Comment:
shouldn't we maybe just add a function in common like:
```r
.getSeedJASP <- function(options) {
  if (is.list(options) && c("setSeed", "seed") %in% names(options)) {
    if (isTRUE(options[["setSeed"]]))
      options[["seed"]]
  }
```
and then call this one all over? That way the seed calling is all located in one place.

Reply:
added